content/shmem_wait.tex

```diff
@@ -52,15 +51,13 @@
 
 \apidescription{ 
     \FUNC{shmem\_wait} and \FUNC{shmem\_wait\_until} wait for \VAR{ivar} to be
-    changed by a remote write or an atomic operation issued by a different \ac{PE}.
+    changed by a write or an atomic operation issued by a \ac{PE}.
     These  routines can be used for point-to-point direct synchronization.  A call
-    to \VAR{shmem\_wait} does not return until some other \ac{PE} writes  a value,
+    to \FUNC{shmem\_wait} does not return until a \ac{PE} writes a value,
     not equal to \VAR{cmp\_value}, into \VAR{ivar} on the waiting \ac{PE}.  A call
-    to \FUNC{shmem\_wait\_until} does not return until some  other \ac{PE} changes
+    to \FUNC{shmem\_wait\_until} does not return until a \ac{PE} changes
     \VAR{ivar} to satisfy the condition implied by \VAR{cmp} and \VAR{cmp\_value}.
-    This mechanism is useful when a \ac{PE} needs to tell another \ac{PE} that it
-    has completed some action.  The \FUNC{shmem\_wait}  routines return when
-    \VAR{ivar} is no  longer  equal  to \VAR{cmp\_value}. The
+    The \FUNC{shmem\_wait}  routines return when \VAR{ivar} is no  longer  equal  to \VAR{cmp\_value}. The
```

Comment:
Was this sentence removed intentionally?

> This mechanism is useful when a \ac{PE} needs to tell another \ac{PE} that it has completed some action.

Reply:
Yes.

Reply:
The changes were read.

Reply:
Ok.